docs/dev-setup.md

```diff
@@ -123,13 +123,33 @@ After you have cloned both repositories, your folder structure should look like:
 |-- src/cluster-api (run `tilt up` here)
 ```
 
+Checkout a stable CAPI release, for example:
+
+```bash
+git checkout release-0.3
+```
+
+Install kustomize
+
+```bash
+cluster-api$ make kustomize
+hack/ensure-kustomize.sh
+fetching kustomize@3.9.1
```

Comment:
Nit: maybe ?
```suggestion
```bash
pushd cluster-api
make kustomize
hack/ensure-kustomize.sh
```

Reply:
There was a mix up of the commands and their output. 
removed the outputs